### PR TITLE
test: Add tests for login and project overview (DEV-3981)

### PR DESCRIPTION
**Pull request by seakayone**
- **Extract commonly used constants**
- **add project overview page test**
- **add login success workflow test**
- **add documentation for credentials to README**

### CODE REVIEW

**Review by siers — APPROVED** [review]
Superficially looks alright.

**Review by mpro7 — APPROVED** [review]
Looks good, please check my suggestions below.

File: k6/README.md [hunk 1]
```diff
@@ -19,6 +19,26 @@ You can run the tests using the following command:
 k6 run tests/<test_script.js>
 ```
 
+### Tests Which Require Authentication
+
+Test that require authentication need to be run with the correct environment variables set.
```
[the remark this answers is not in the document]
Comment (seakayone) [reply k6/README.md:24]:
```suggestion
Tests that require authentication need to be run with the correct environment variables set.
```

File: k6/justfile [hunk 2]
```diff
@@ -1,5 +1,7 @@
 appUrl := "https://app.dev.dasch.swiss"
 testToRun := "first-page-load"
+#k6_browser_headless := "false"
```
[the remark this answers is not in the document]
Comment (seakayone) [reply k6/justfile:3]: No, I have introduced the variable to be commented in for local development purposes.

**Pull request merged**